sections/experiments/results.tex

```diff
@@ -12,7 +12,7 @@ \subsubsection{LDOS-CoMoDa}
 These also achieve the highest MAP@K values but do not have as low prediction errors as the \texttt{CAMF} methods.
 \texttt{NMF} generally perform a little worse in all metrics compared to \texttt{SVD}, except for NDCG.
 The \texttt{DeepWalk + kNN} method has high prediction errors and low Precision@10 and MAP@10 values but has the second-highest Recall@10 and one of the highest NDCG value.
-Finally, we have the methods that do not handle rating predictions but only do top K recommendations, so these do not have RMSE and MAE values, since they work with interactions rather than ratings.
+Finally, we have the methods that do not handle rating predictions but only do top-N recommendations, so these do not have RMSE and MAE values, since they work with interactions rather than ratings.
```

Comment:
```suggestion
Finally, we have the methods that do not handle rating predictions but only do top-$N$ recommendations, so these do not have RMSE and MAE values, since they work with interactions rather than ratings.
```